action.yml

```diff
@@ -26,12 +26,8 @@ runs:
     - name: Push Gem
       id: push-gem
       shell: bash
-      env:
-        # Expects GEM_HOST and GEM_HOST_API_KEY to be set
-        INPUT_PACKAGE_GLOB: ${{ inputs.package-glob }}
-        INPUT_RELEASE: ${{ inputs.release }}
-        INPUT_PRE_RELEASE: ${{ inputs.pre-release }}
-        INPUT_TAG_RELEASE: ${{ inputs.tag-release }}
       run: |
         PATH="${{ github.action_path }}:$PATH"
-        gem-push-action
+        args=""
+        '${{ inputs.tag }}' && args="$args -t"
```

Comment:
> '${{ inputs.tag }}'

I reckon unsanitised execution of user input might surprise us in the future. I see it's safe here (since someone can already just add whatever steps they like to a branch's workflow), but I worry about someone copying it to someplace less safe

How about plain old `[ '${{ input.tag }}' = 'true' ] && args="$args -t"`?

Reply:
Is is not actually executed, as it expands to this when run under bash 😛 

```
  'true' && args="$args -t"
```

But it is all a little too cute maybe, going with plain old 👍🏻